flow/SimpleOpt.h

```diff
@@ -280,7 +280,11 @@ enum _ESOFlags {
 	SO_O_ICASE_WORD = 0x0400,
 
 	/*! Case-insensitive comparisons for all arg types */
-	SO_O_ICASE = 0x0700
+	SO_O_ICASE = 0x0700,
+
+	/*! Transfer all hyphens of users' input flags and internal flags to underscores excetp leading hyphens
```

Comment:
```suggestion
	/*! Treat all hyphens from flag names as underscores except leading hyphens
```

Reply:
That's a more clear statement. Thank you!